### PR TITLE
Airdrop with mutation

### LINKED ISSUE #23: Airdrop page

### PR DESCRIPTION
**Pull request by kompotkot**
## Related issues
closes #23

### CODE REVIEW

File: pages/airdrop/index.tsx [hunk 1]
```diff
@@ -1,46 +1,83 @@
...
 
 const Airdrop = () => {
   const toast = useToast()
-  const [searchingAddress, setSearchingAddress] = useState('')
-  const [claimedStatus, setClaimedStatus] = useState('')
+  const [claimantAddress, setClaimantAddress] = useState('')
+  const [claimantEmail, setClaimantEmail] = useState('')
+  const [claimantDiscord, setClaimantDiscord] = useState('')
+
+  const web3 = new Web3()
```
Comment (kompotkot) [inline pages/airdrop/index.tsx:18]: Better to define it once at app or layout

File: pages/airdrop/index.tsx [hunk 2]
```diff
@@ -1,46 +1,83 @@
...
+
+  const ENTITY_API = process.env.NEXT_PUBLIC_ENTITY_API_URL
```
Comment (kompotkot) [inline pages/airdrop/index.tsx:20]: Organize all configs and env vars at one place in `src/constants.ts`

File: pages/airdrop/index.tsx [hunk 3]
```diff
@@ -1,46 +1,83 @@
...
+  const WHITELIST_EVENT_COLLECTION_ID = process.env.NEXT_PUBLIC_WHITELIST_EVENT_COLLECTION_ID
+
+  const onSuccess = (data: any) => {
+    toast({
```
Comment (kompotkot) [inline pages/airdrop/index.tsx:24]: This is is duplication of https://github.com/bugout-dev/portal/blob/2cf35cf045cd1e22de2fab05c10da925af088278/pages/terminus/index.tsx#L67
Can we have toas component which we will be reuse with different argument `message`?

File: pages/airdrop/index.tsx [hunk 4]
```diff
@@ -1,46 +1,83 @@
...
+    position: 'top',
+  })}
+
 
-  const onSuccess = () => {
-    // Handle successful request
-  }
   const onError = (error: any) => {
-    console.log(error?.message)
     toast({
```
Comment (kompotkot) [inline pages/airdrop/index.tsx:36]: same with other toast

**Review by kompotkot — COMMENTED** [review]
rc

**Review by Anton-Mushnin — APPROVED** [review]
lg

**Pull request merged**